### PR TITLE
⚙️ Settings accordion

### PR DESCRIPTION
**Pull request by vasuchal**
## Description
The AccordionButton Component from ReachUI is wrapped to meet our style requirements.

## Screenshots

When button is closed
Screen Shot 2021-03-31 at 2 41 18 PM

When button is open
Screen Shot 2021-03-31 at 2 41 40 PM

Screen Shot 2021-03-31 at 2 44 44 PM

### CODE REVIEW

File: src/components/ui/ListEntry.js [hunk 1]
```diff
@@ -46,13 +46,14 @@ const TextContainer = styled.div`
...
 
+// Forwarding ref here seems necessary for Combobox highlighting to work
 const ListEntry = React.forwardRef(
   ({ leftIcons, primaryText, secondaryText, rightIcons, ...props }, ref) => (
     <ListContainer ref={ref} {...props}>
       <Icons>{leftIcons}</Icons>
       <TextContainer>
-        <PrimaryText>{primaryText}</PrimaryText>
-        <SecondaryText>{secondaryText}</SecondaryText>
+        <PrimaryText className="primaryText">{primaryText}</PrimaryText>
```
Comment (Archna-1) [inline src/components/ui/ListEntry.js:55]: I don't think there's anything wrong with this, but I was wondering how having the className changes this.

Comment (Qrtn) [reply src/components/ui/ListEntry.js:55]: @vasuchal

Comment (Qrtn) [reply src/components/ui/ListEntry.js:55]: Good question!!

**Review by Archna-1 — APPROVED** [review]
LGTM!

File: src/components/ui/ListEntry.js (hunk 1 above)
Comment (vasuchal) [reply src/components/ui/ListEntry.js:55]: We need that so that we can override styles on ListEntry in the AccordionButton using CSS selectors.

**Review by laurenho025 — APPROVED** [review]
looks good 😍

**Review by rxun — APPROVED** [review]

**Pull request merged**